### PR TITLE
[PERFSCALE-3547] Parametrize installation variables

### PR DESCRIPTION
**Pull request by krishvoor**
With this PR, all relevant variables are properly parametrized.

### CODE REVIEW

**Comment by josecastillolema** [comment]
cc @rsevilla87

File: ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh [hunk 1]
```diff
@@ -28,15 +28,15 @@ ssh_public_key_file: ~/.ssh/id_rsa.pub
...
-bastion_vlaned_interface: ens1f1
+bastion_vlaned_interface: $BASTION_VLANED_INTERFACE
 setup_bastion_gogs: false
 setup_bastion_registry: false
 use_bastion_registry: false
-controlplane_lab_interface: eno1np0
+controlplane_lab_interface: $CONTROLPLANE_LAB_INTERFACE
 controlplane_pub_network_cidr:
 controlplane_pub_network_gateway:
 jumbo_mtu: false
```
Comment (rsevilla87) [inline ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh:42]: Can we parametrize this one as well?

**Comment by josecastillolema** [comment]
@rsevilla87 correct me if I am wrong, but I think this would be the proper PR to implement the copy ssh key logics.
So, after the deployments finishes, we need to do (pseudocode warning):
```
if cloud != 31 # if we are deploying a cloud different than the BM CPT lab one
   scp key to the cloud bastion host
fi
```

File: ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh (hunk 1 above)
Comment (krishvoor) [reply ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh:42]: Sure, just to be clear are you referring to the below?
```
controlplane_network
controlplane_network_prefix
controlplane_pub_network_cidr
controlplane_pub_network_gateway
```

**Comment by rsevilla87** [comment]
> @rsevilla87 correct me if I am wrong, but I think this would be the proper PR to implement the copy ssh key logics. So, after the deployments finishes, we need to do (pseudocode warning):
> 
> ```
> if cloud != 31 # if we are deploying a cloud different than the BM CPT lab one
>    scp key to the cloud bastion host
> fi
> ```

I think its better to address it in a different PR so we can move forward with this one

File: ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh (hunk 1 above)
Comment (rsevilla87) [reply ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh:42]: no, just jumbo_mtu

**Comment by rsevilla87** [comment]
> I suggest at least 2 rehearsals:
> 
>     * BM CPT lab (cloud31)
> 
>     * cloud17

Doing a rehearsal in cloud17 requires another commit, is that you you meant?

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.17-latest-x86-deploy-6nodes

**Comment by krishvoor** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Comment by krishvoor** [comment]
>The following jobs are not rehearsable without the network-access-rehearsals-ok, and approved labels present on this PR. This is due to the restrict_network_access field being set to false. The network-access-rehearsals-ok label can be added by any openshift org member other than the PR's author by commenting: /pj-rehearse network-access-allowed:

**Comment by krishvoor** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Comment by krishvoor** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Comment by rsevilla87** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.17-latest-x86-deploy-sno

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-4.16-latest-x86-deploy-sno

**Pull request merged**